### PR TITLE
split FileAPI/idlharness tests into any.js

### PR DESCRIPTION
**Pull request by lucacasonato**

### CODE REVIEW

File: FileAPI/idlharness.html [hunk 1]
```diff
@@ -27,10 +27,7 @@ <h1>File API automated IDL tests</h1>
         ['dom', 'html', 'url'],
         idl_array => {
           idl_array.add_objects({
-            Blob: ['new Blob(["TEST"])'],
-            File: ['new File(["myFileBits"], "myFileName")'],
             FileList: ['document.querySelector("#fileChooser").files'],
```
Comment (inexorabletash) [inline FileAPI/idlharness.html:30]: Nit: Inconsistent use of trailing comma.

Should be harmless with modern JS, but consistency is nice. :)

**Review by inexorabletash — APPROVED** [review]

File: FileAPI/idlharness.html (hunk 1 above)
Comment (lucacasonato) [inline FileAPI/idlharness.html:30]:
```suggestion
            FileList: ['document.querySelector("#fileChooser").files']
```

**Comment by lucacasonato** [comment]
@inexorabletash Could you merge this? I don't have WPT write access.

**Comment by inexorabletash** [comment]
Done!

**Comment by lucacasonato** [comment]
Thanks!

**Pull request merged**